modules/buzzoolaBidAdapter.md

```diff
@@ -0,0 +1,71 @@
+# Overview
+
+```
+Module Name:  Buzzoola Bid Adapter
+Module Type:  Bidder Adapter
+Maintainer: devteam@buzzoola.com
+```
+
+# Description
+
+Connects to Buzzoola exchange for bids.
+
+Buzzoola bid adapter supports Banner and Video (instream and outstream).
+
+# Test Parameters
+```
+var adUnits = [
+    // Banner adUnit
+    {
+        code: 'banner-div',
+        mediaTypes: {
+            banner: {
+                sizes: [[240, 400], [300, 600]],
+            }
+        },
+        bids: [{
+            bidder: 'buzzoola',
+            params: {
+                placementId: 417846
+            }
+        }]
+    },
+    // Video instream adUnit
+    {
+        code: 'video-instream',
+        mediaTypes: {
+            video: {
```

Comment:
I think you forgot the add the context property to the video object.

Reply:
added